dataikuapi/dss/recipe.py

```diff
@@ -76,15 +76,30 @@ def run(self, job_type="NON_RECURSIVE_FORCED_BUILD", partitions=None, wait=True,
         :return: the :class:`dataikuapi.dss.job.DSSJob` job handle corresponding to the built job
         :rtype: :class:`dataikuapi.dss.job.DSSJob`
         """
+        project = self.client.get_project(self.project_key)
 
-        settings = self.get_settings()
-        output_refs = settings.get_flat_output_refs()
+        continuous_recipes_names = set(activity.recipe_id for activity in project.list_continuous_activities())
```

Comment:
Wondering how this scales for very large flows / whether it would be better to add to the backend API an indication that some recipe is continuous.
Another possibility could be to add the check on the backend side on the start call.